packages/sdk/src/contracts/Operator.ts

```diff
@@ -575,6 +575,8 @@ export class Operator {
     async voteOnFlag(sponsorshipAddress: EthereumAddress, targetOperator: EthereumAddress, kick: boolean): Promise<void> {
         const voteData = kick ? VOTE_KICK : VOTE_NO_KICK
         await this.connectToContract()
+        // estimateGas throws if transaction would fail, ignore the return value
```

Comment:
I suppose we could handle the case where the estimation is higher than 1.3M... since that estimate is conservative (higher than expected) probably we could even not send the transaction. I'll add this logic.